pkg/controllers/managementuser/resourcequota/resource_quota_sync.go

```diff
@@ -450,9 +450,11 @@ func (c *SyncController) getResourceLimitToUpdate(ns *corev1.Namespace) (*corev1
 		return convertPodResourceLimitToLimitRangeSpec(updatedLimit)
 	} else if nsLimit != nil {
 		return convertPodResourceLimitToLimitRangeSpec(nsLimit)
+	} else if projectLimit != nil {
+		return convertPodResourceLimitToLimitRangeSpec(projectLimit)
```

Comment:
Question on this code - 2.5 does not appear to have your changes which changed how the https://github.com/rancher/rancher/commit/fa9806e37e0381f620c0632878986d9a43bc7eaf). That may cause this new code to behave differently than we saw in 2.6 - do we want to backport the other fix as well?

Reply:
Yes, I had looked at the other issue first, and, strangely, could not reproduce it under v2.5. I will try again though.
In any case, this fix is not really connected with the other issue, so their sequential order is not relevant.

Reply:
The reason why I ask here is that other fix (if I was looking at it right) affected the output of `getProjectContainerDefaultLimit`. So in scenarios where the merged value was nil and the namespace value was nil, it would be using the output of that function - which is different for 2.6 because of that fix. The case where it would matter (nil quota but non-nil default limits) is pretty narrow but I think still worth double checking (at least over github).

Reply:
Yeah, good point. I will double-check. I just tried v2.5.12 again, and not having resource quotas does not prevent container default limits from being propagated, which is interesting, will investigate.